Framework/DataHandling/test/LoadSampleShapeTest.h

```diff
@@ -0,0 +1,174 @@
+#ifndef LOAD_SHAPETEST_H_
+#define LOAD_SHAPETEST_H_
+
+#include "MantidAPI/FrameworkManager.h"
+#include "MantidAPI/AnalysisDataService.h"
+#include "MantidAPI/Sample.h"
+#include "MantidDataHandling/LoadSampleShape.h"
+#include "MantidDataHandling/LoadInstrument.h"
+#include "MantidGeometry/Objects/MeshObject.h"
+#include "MantidKernel/OptionalBool.h"
+#include "MantidTestHelpers/ComponentCreationHelper.h"
+#include "MantidTestHelpers/WorkspaceCreationHelper.h"
+#include <cxxtest/TestSuite.h>
+
+using namespace Mantid;
+using namespace Mantid::API;
+using namespace Mantid::DataHandling;
+using namespace Mantid::DataObjects;
+using namespace Mantid::Geometry;
+
+class LoadSampleShapeTest : public CxxTest::TestSuite {
+public:
+  static LoadSampleShapeTest *createSuite() {
+    return new LoadSampleShapeTest();
+  }
+  static void destroySuite(LoadSampleShapeTest *suite) { delete suite; }
+
+  void testName() { TS_ASSERT_EQUALS(loadShape.name(), "LoadSampleShape"); }
+
+  void testVersion() { TS_ASSERT_EQUALS(loadShape.version(), 1); }
+
+  void testInit() {
```

Comment:
Rather pointless test, also this is the only test that uses the class variableloadShpae, all others define there own, why? consistency.  This looks just copied and pasted without understanding.